java/build.gradle

```diff
@@ -17,6 +17,9 @@ subprojects {
     // minimal java compatibility level
     plugins.withId('java') {
         sourceCompatibility = targetCompatibility = "11"
+        dependencies {
```

Comment:
same - lets keep this PR only for bumping version, i'm not sure if we need these dependencies to be required by all submodules